src/payment/index.ts

```diff
@@ -173,7 +173,7 @@ export class Payment extends Config {
   private validateResponseHash = (data: any) => {
```

Comment:
After reading the other docs from Urway, specifically recurring. this should be moved to config. We also, going to need it in the subscription. what do you think ?

Reply:
I never saw subscription config, can you give me access to docs or related links?

I also think it's good move to keep it into config class since we may need it in other subclasses.